internal/it/runtime/arch.go

```diff
@@ -0,0 +1,13 @@
+package runtime
+
+import "runtime"
+
+func Is32BitArch() bool {
+	archs := []string{"386", "arm", "mips", "mipsle"}
+	for _, arch := range archs {
+		if runtime.GOARCH == arch {
+			return true
+		}
+	}
```

Comment:
This works, but would using `switch` be simpler here?
```
switch runtime.GOARCH {
case "386":
	return true
// ...
}
return false
```

Reply:
Thanks of course, sorry for that